api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
Is this what you mean if I understand correctly.
<img width="888" alt="screen shot 2016-02-18 at 12 22 34 am" src="https://cloud.githubusercontent.com/assets/10450478/13120829/88ee2a6c-d5d5-11e5-9f22-fec64decaad8.png">

My expectation was I do not want to change the ordering of orders which doesn't have sequence number. But the Util method will change that. It will put them at beginning / end. Is it not wrong?